rootfs/etc/cont-init.d/03-config.sh

```diff
@@ -112,6 +112,14 @@ DB_USERNAME=${DB_USER}
 DB_PASSWORD=${DB_PASSWORD}
 EOL
 
+# Enable first run wizard if db is empty
+counttables=$(echo 'SHOW TABLES' | ${dbcmd} "$DB_NAME" | wc -l)
+if [ "${counttables}" -eq "0" ]; then
+  echo "Enabling First Run Wizard..."
+  echo "INSTALL=user,finish">> ${LIBRENMS_PATH}/.env
```

Comment:
Can you add a comment with a link to the official docs about the `INSTALL` var please?

Reply:
Is the `INSTALL` var changed in `.env` when wizard is completed?

Reply:
@crazy-max no official docs.

INSTALL=true
Run all steps of the installer

INSTALL=<comma separated steps>
Run only the specified steps.

Once the installer is finalized, the .env is re-written to remove INSTALL and update any other vars.

I want to add some first run steps to the wizard, such as scanning a network for snmp devices and adding them.